### PR TITLE
Fz83 p 5/hero

### PR DESCRIPTION
**Pull request by Decomplosed**
check how it works for now

### CODE REVIEW

File: assets/style/style.css [hunk 1]
```diff
@@ -80,25 +80,110 @@ h2 {
...
+    0% {background-image: url("../images/gdansk.jpg");}
+    33% {background-image: url("../images/sopot2.png");}
+    66% {background-image: url("../images/gdynia.jpg");}
+    100% {background-image: url("../images/gdansk.jpg");}
+}
+
+.hero-button{
+    text-shadow: 1px 1px 2px #0a0a0a;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    width: 20rem;
```
Comment (bartzaw) [inline assets/style/style.css]: this 'rem' thing, @sebastian-kreft  mentioned before that it shouldn't be used as a regular thing. Sebastian, can you elaborate? We can arrange call via slack with all 3 of us as it's not clear enough for me why it should not be used as often.

File: assets/style/style.css [hunk 2]
```diff
@@ -80,25 +80,110 @@ h2 {
...
+    padding-top: 10px;
+    padding-bottom: 10px;
+    display: inline-block;
+    margin: 0 auto;
+    text-decoration: none;
+    border: 3px solid white;
+    border-radius: 20px;
+    color: black;
```
Comment (bartzaw) [inline assets/style/style.css]: please use only colors using hex or rgb

File: index.html [hunk 3]
```diff
@@ -37,13 +34,13 @@
         <div class="hero">
             <div class="hero-content">
                 <div class="hero-title">
-                    <h1>Lorem ipsum dolor sit amet, consectetur adipisicing elit.</h1>
-                    <h2>Lorem ipsum dolor sit amet</h2>
-                    <h2>Lorem ipsum dolor sit amet</h2>
-                </div>
-                <div class="cta-buttons">
-                    <button><span class="fas fa-arrow-alt-circle-right"></span> Zapisz się!</button>
+                    <h1>3VENT CITY - ENJOY 3CITY</h1>
```
Comment (bartzaw) [inline index.html]: Please do not use capital letters in html code, should be replaced by text-transform in css

**Review by bartzaw — CHANGES_REQUESTED** [review]
Please check out my comments, I

File: assets/js/main.js [hunk 4]
```diff
@@ -11,8 +11,10 @@ $(document).ready(function(){
         $(".navigation-list").toggleClass('hide');
     });
 });
-
+/*
```
Comment (icelandico) [inline assets/js/main.js]: I'm not sure leaving commented code is good idea?

File: assets/style/style.css (hunk 1 above)
Comment (icelandico) [reply assets/style/style.css]: Maybe for element width it's better to use `%` or `px`.

**Review by icelandico — APPROVED** [review]

**Review by bartzaw — APPROVED** [review]

File: assets/js/main.js [hunk 5]
```diff
@@ -10,4 +10,11 @@ $(document).ready(function(){
         event.preventDefault();
         $(".navigation-list").toggleClass('hide');
     });
-});
\ No newline at end of file
+});
+/*
+$('.carousel').carousel({
```
Comment (sebastian-kreft) [inline assets/js/main.js]: Don't leave such comments in the code, it's untidy :P

**Pull request merged**